Header header
int32 width
int32 height
uint8[] data
int64 hwTimestamp
float64 gain
int32 exposureTime
int32 pixelClock
